schemas/schemas/infoAssessment.json

```diff
@@ -166,6 +166,11 @@
                             }
                         }
                     }
+                },
+                "showClosedAssessment": {
+                    "description": "If true, prevents the assessment from being viewed once it has been closed",
```

Comment:
Should this be

`If *false*, prevents the assessment from being viewed once it has been closed`?

Otherwise the name seems confusing

Reply:
Oh yes! Good catch!

I reworded it completely.